source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,161 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. versionadded:: 2.3.0
+
+.. figure:: ../../_static/controllers/native_instruments_traktor_kontrol_s3.jpg
```

Comment:
Please add labels to the controls (ideally use an SVG if possible) or remove.

Reply:
removed